### ISSUE TITLE
PANO: HA Panoramas cause snapshot init to fail

### ISSUE THREAD

**Issue by jeffkala** [batfish/batfish#8620]
**Describe the bug and expected behavior**
HA Panoramas cause snapshot init to fail. As soon as one of the configs is removed the snapshot completes success. 

```python
>>> bf.init_snapshot(SDIR, name="test1", overwrite=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/pybatfish/client/session.py", line 823, in init_snapshot
    upload, name=name, overwrite=overwrite, extra_args=extra_args
  File "/usr/local/lib/python3.7/site-packages/pybatfish/client/session.py", line 952, in _init_snapshot
    return self._parse_snapshot(name, background, extra_args)
  File "/usr/local/lib/python3.7/site-packages/pybatfish/client/session.py", line 1210, in _parse_snapshot
    work_item, self, background=background, extra_args=extra_args
  File "/usr/local/lib/python3.7/site-packages/pybatfish/client/workhelper.py", line 152, in execute
    prefix="..." if log_file_msg else "",
pybatfish.exception.BatfishException: Work terminated abnormally
work_item: {"containerName": "Example1", "id": "b587cb91-efea-47d4-89e6-bab3f16f3480", "requestParams": {"testrig": "test1", "si": "", "sv": "", "initinfo": ""}, "testrigName": "test1"}

<omitted bunch of MISCELLANEOUS logs>

Exception in container:cd656901-f625-4eee-9d62-bfd94daea934, testrig:6ac2f7fd-ee9c-4cbc-a74b-ff7edbedbf8c; exception:org.batfish.common.BatfishException: Duplicate hostname: 001701011257
	at org.batfish.job.ConvertConfigurationResult.applyTo(ConvertConfigurationResult.java:70)
	at org.batfish.job.ConvertConfigurationResult.applyTo(ConvertConfigurationResult.java:17)
	at org.batfish.job.BatfishJobExecutor.handleJobResult(BatfishJobExecutor.java:220)
	at org.batfish.job.BatfishJobExecutor.executeJobs(BatfishJobExecutor.java:142)
	at org.batfish.job.BatfishJobExecutor.runJobsInExecutor(BatfishJobExecutor.java:70)
	at org.batfish.main.Batfish.convertConfigurations(Batfish.java:777)
	at org.batfish.main.Batfish.getConfigurations(Batfish.java:877)
	at org.batfish.main.Batfish.serializeIndependentConfigs(Batfish.java:2327)
	at org.batfish.main.Batfish.run(Batfish.java:2089)
	at org.batfish.main.Driver.lambda$runBatfish$0(Driver.java:247)
	at java.base/java.lang.Thread.run(Thread.java:829)
{"answerElements":[{"class":"org.batfish.common.BatfishException$BatfishStackTrace","answer":["org.batfish.common.BatfishException: Duplicate hostname: 001701011257","   at org.batfish.job.ConvertConfigurationResult.applyTo(ConvertConfigurationResult.java:70)","   at org.batfish.job.ConvertConfigurationResult.applyTo(ConvertConfigurationResult.java:17)","   at org.batfish.job.BatfishJobExecutor.handleJobResult(BatfishJobExecutor.java:220)","   at org.batfish.job.BatfishJobExecutor.executeJobs(BatfishJobExecutor.java:142)","   at org.batfish.job.BatfishJobExecutor.runJobsInExecutor(BatfishJobExecutor.java:70)","   at org.batfish.main.Batfish.convertConfigurations(Batfish.java:777)","   at org.batfish.main.Batfish.getConfigurations(Batfish.java:877)","   at org.batfish.main.Batfish.serializeIndependentConfigs(Batfish.java:2327)","   at org.batfish.main.Batfish.run(Batfish.java:2089)","   at org.batfish.main.Driver.lambda$runBatfish$0(Driver.java:247)","   at java.base/java.lang.Thread.run(Thread.java:829)",""]}],"status":"FAILURE","summary":{"numFailed":0,"numPassed":0,"numResults":0}
```

**Additional context**
Panoramas in an active/standby, when both configs are provided in a init snapshot the failure below occurs.  The anonymized configs are provided.
[ny4-anonymized.txt](https://github.com/batfish/batfish/files/10211909/ny4-anonymized.txt)
[ch2-anonymized.txt](https://github.com/batfish/batfish/files/10211912/ch2-anonymized.txt)

**TLDR**
Does Batfish support HA panorama setups?  I didn't see anything in the documentation mentioning that it does or does not.

**Comment by dhalperi** [comment]
We haven't run into this before.

`diff`-ing the two files, it looks like this is the key bit:

```
93085,93086c93077
< set deviceconfig high-availability peer ip-address 10.20.122.39
< set deviceconfig high-availability peer serial 017507002662
---
> set deviceconfig high-availability peer ip-address 10.123.112.31
93087a93079
> set deviceconfig high-availability election-option priority secondary
93090d93081
< set deviceconfig high-availability election-option priority primary
```

This makes me think we should just skip converting any of the managed devices for the secondary. Seems a relatively easy fix, if you want to try it out :)
